tests/server/test_server_validation.py

```diff
@@ -20,6 +20,19 @@ def test_with_validator(both_fake_remote_clients):
     assert validator.valid
 
 
+def test_with_validator_skip_optional(both_fake_remote_clients):
+    from optimade.server.main_index import app
+
+    validator = ImplementationValidator(
+        client=both_fake_remote_clients,
+        index=both_fake_remote_clients.app == app,
+        run_optional_tests=True,
```

Comment:
```suggestion
        run_optional_tests=False,
```
The default value for run_optional_tests is already True, so otherwise this test does nothing different from the test before it.